Plugins/Flow.Launcher.Plugin.Program/Programs/UWP.cs

```diff
@@ -226,10 +233,31 @@ private static IEnumerable<Package> CurrentUserPackages()
             }
             else
             {
-                return new Package[] { };
+                return new Package[]
+                {
+                };
             }
         }
 
+        private static List<FileSystemWatcher> _watchers = new();
+
+        private static void GenerateWatcher(string path)
+        {
+            var watcher = new FileSystemWatcher(path);
+            watcher.Created += static (_, _) => Task.Run(Main.IndexUwpPrograms);
+            watcher.Deleted += static (_, _) => Task.Run(Main.IndexUwpPrograms);
+            watcher.EnableRaisingEvents = true;
+        }
```

Comment:
This is not used, remove?

Reply:
Huh probably I forget something. I will check again.

Reply:
Removed